sidecar-pg.Dockerfile

```diff
@@ -0,0 +1,75 @@
+FROM datamon-binaries as base
+
+# dist-alike during development/debug
+RUN cp /stage/usr/bin/datamon /usr/bin/datamon
+ADD ./hack/fuse-demo/datamon.yaml /root/.datamon/datamon.yaml
+
+# Build the dist image
+# ??? do not need fuse and sudo in case of bundle download instead of mount
+FROM postgres:11.5
```

Comment:
I am not following clearly here why we build a postgres from scratch (and a lot of pg-related scripting) when we can probably be content with a standard postgres or postgis image, possibly installing some extra postgres addons. Does this come from a constraint on the version?
Using a standard docker image, the use-case corresponding to "mounting from a dump" becomes trivial, since the image startup script automatically provision any dump(s) given in its init.d.

Reply:
Oops. Was mislead here. Please ignore my comment